### PR TITLE
#738 -  Fixed markdown linting violations from landing zone v2 readme document

### PR DESCRIPTION
**Pull request by jacyang2010**
This PR is to fix some minor markdown issues from the landingzone v2 readme.md.
See #738

### CODE REVIEW

**Comment by jacyang2010** [comment]
Just a tiny doc pr to make a start here. Please feel free to review it. @fmichaelobrien and @cartyc

**Review by fmichaelobrien — COMMENTED** [review]
Nice, the indenting on that section has been an issue for part of the year - thank you
will hold off on the PR submit until the release - which is still ongoing as of 1600 right now.
Will revisit on monday Jackson for main PRs

**Comment by jacyang2010** [comment]
I see that and please let me know when it can be merged after release. @fmichaelobrien

**Comment by fmichaelobrien** [comment]
All good, Jackson the changes look good - I will load the branch to render a quick check
Another release just came in and likely we will check on comms for the final code freeze

https://github.com/GoogleCloudPlatform/pubsec-declarative-toolkit/pull/735#pullrequestreview-1748423863

**Comment by fmichaelobrien** [comment]
Jackson, good morning
- pending lint fixes
- pending (optional) link to a github issue ID
- added full team reviewers
- see ongoing/expanding contributor onboarding agreement - https://github.com/GoogleCloudPlatform/pubsec-declarative-toolkit/wiki/Contributor-Onboarding-Agreement#contributor-onboarding-agreement

**Comment by fmichaelobrien** [comment]
Reviewing the branch - can it be merged with the latest main updates (10 behind)
https://github.com/GoogleCloudPlatform/pubsec-declarative-toolkit/tree/gh333-yw-landingzone-v2-readme-doc

File: docs/landing-zone-v2/README.md
Comment (jacyang2010) [reply docs/landing-zone-v2/README.md:642]: Done.

**Review by fmichaelobrien — APPROVED** [review]
/lgtm

**Comment by fmichaelobrien** [comment]
thanks for the split off of the kubectl delete - script section in 
https://github.com/GoogleCloudPlatform/pubsec-declarative-toolkit/issues/739
and merging with main

**Comment by jacyang2010** [comment]
> Jackson, can we spawn off a separate issue/pr for the lz delete with minimal testing. I feel it needs a separate pr like the following
> 
> #593
> 
> We can keep this pr on readme formatting
Done

**Comment by fmichaelobrien** [comment]
Sounds good, I also inadvertently caused a 2nd merge conflict with Jackson's 722.  I was thinking of keeping 744 unmerged until after his so I would take on the merge conflict.  All good, next time I will put a heads up comment in the PR.
https://github.com/GoogleCloudPlatform/pubsec-declarative-toolkit/pull/744
https://github.com/GoogleCloudPlatform/pubsec-declarative-toolkit/pull/722/files

**Comment by jacyang2010** [comment]
> Sounds good, I also inadvertently caused a 2nd merge conflict with Jackson's 722. I was thinking of keeping 744 unmerged until after his so I would take on the merge conflict. All good, next time I will put a heads up comment in the PR. #744 https://github.com/GoogleCloudPlatform/pubsec-declarative-toolkit/pull/722/files

Rebased to latest main.

**Comment by jacyang2010** [comment]
@cartyc do you mind review and approve this PR as the second reviewer?

**Review by fmichaelobrien — DISMISSED** [review]
/lgtm
However, we need to wait until the current release is merged and the code freeze on main is lifted - before merging this PR

https://github.com/GoogleCloudPlatform/pubsec-declarative-toolkit/pull/741

File: docs/landing-zone-v2/README.md
Comment (davelanglois-ssc) [inline docs/landing-zone-v2/README.md:21]: I'm wondering why these are being removed from the table of content.

also, has anybody tried replacing the entire table of content with just "[[_TOC_]]" and let github render it.

Comment (alaincormier-ssc) [reply docs/landing-zone-v2/README.md:21]: GitHub's built-in Outline works well, even better for navigation because it's pinned on the side as you click/scroll

image

**Comment by Michael-McAuley-SSC** [comment]
Unclassified | Non classifié

I have tried to use TOC in github. – no bueno.

**Review by obriensystems — APPROVED** [review]
/lgtm

**Review by fmichaelobrien — APPROVED** [review]
/lgtm

**Comment by fmichaelobrien** [comment]
Merge #738 or#739 first and fix any merge conflict

**Review by davelanglois-ssc — APPROVED** [review]

**Pull request merged**
